tests/msc2716_test.go

```diff
@@ -465,7 +463,7 @@ func TestBackfillingHistory(t *testing.T) {
 		})
 
 		t.Run("Historical messages are visible when already joined on federated server", func(t *testing.T) {
-			t.Skip("Skipping until federation is implemented")
+			//t.Skip("Skipping until federation is implemented")
```

Comment:
This is dependent on https://github.com/matrix-org/synapse/pull/10498 and can remove or leave it depending on that PR status when we merge this.